mailbox/api/src/main/java/org/apache/james/mailbox/indexer/ReIndexingExecutionFailures.java

```diff
@@ -62,16 +62,16 @@ public final int hashCode() {
         }
     }
 
-    private final List<ReIndexingFailure> failures;
+    private final List<ReIndexingFailure> messageFailures;
     private final List<MailboxId> mailboxFailures;
 
-    public ReIndexingExecutionFailures(List<ReIndexingFailure> failures, List<MailboxId> mailboxFailures) {
-        this.failures = failures;
+    public ReIndexingExecutionFailures(List<ReIndexingFailure> messageFailures, List<MailboxId> mailboxFailures) {
+        this.messageFailures = messageFailures;
```

Comment:
Can we have that change in the JSON too?

Reply:
Can do but are the changes going to affect backward compatibility?

Reply:
Not if we have 2 optional parameters in the constructor and mandate only one to be specified.